packages/flutter_tools/lib/src/commands/emulators.dart

```diff
@@ -27,7 +27,7 @@ class EmulatorsCommand extends FlutterCommand {
   final String description = 'List, launch and create emulators.';
 
   @override
-  final List<String> aliases = <String>['emulator'];
+  final List<String> aliases = <String>['emulator', 'simulators', 'simulator'];
```

Comment:
`simulator` is more common than `emulator` on iOS.

Reply:
I'd be consistent with the pluralization. Either both or neither.

Reply:
also, what do you think about aliasing --launch with -d for synergy with flutter run?

Reply:
🤦‍♂️

Reply:
Actually `package:args/args.dart` doesn't support option/flag aliasing.  I'm going to leave this out.  It's not quite the same anyway, without the `launch` verb it's not as clear what you're doing with the device.